doc/languages-frameworks/gnome.xml

```diff
@@ -219,9 +219,9 @@ mkDerivation {
   dontWrapGApps = true;
 
   # Arguments to be passed to `makeWrapper`, only used by qt5’s mkDerivation
```

Comment:
Maybe we should mention `wrapQtAppsHook` instead of `mkDerivation` https://github.com/NixOS/nixpkgs/pull/71089

Reply:
I'm not sure, ideally we'd do that when the change actually happens.
Currently people should be using `mkDerivation`, and there's not code committed that has made it "deprecated". Activity in the regard probably needs to pickup again sometime next year (hopefully in 20.03 or later).